core/platform/email/gae_email_services_test.py

```diff
@@ -43,3 +49,132 @@ def test_sending_email(self):
         messages = self.mail_stub.get_sent_messages(
             to=feconf.ADMIN_EMAIL_ADDRESS)
         self.assertEqual(1, len(messages))
+
+    def test_that_email_bcc_is_sent_when_bcc_admin_is_true(self):
```

Comment:
If you're trying to ensure that an admin is bcc'd then you probably shouldn't be setting ADMIN_EMAIL_ADDRESS as the recipient. Otherwise, how do you know whether they received the email due to the bcc?

Reply:
Done